ext/lists_test.go

```diff
@@ -36,8 +36,8 @@ func TestLists(t *testing.T) {
 		{expr: `[1,2,3,4].slice(0, 10)`, err: "cannot slice(0, 10), list is length 4"},
 		{expr: `[1,2,3,4].slice(-5, 10)`, err: "cannot slice(-5, 10), negative indexes not supported"},
 		{expr: `[1,2,3,4].slice(-5, -3)`, err: "cannot slice(-5, -3), negative indexes not supported"},
-		{expr: `[].flatten() == []`},
-		{expr: `[1,2,3,4].flatten() == [1,2,3,4]`},
+		{expr: `dyn([]).flatten() == []`},
+		{expr: `dyn([1,2,3,4]).flatten() == [1,2,3,4]`},
```

Comment:
From the usability standpoint, wouldn't we want flatten to behave like an identity function if it doesn't contain any nested list? requiring an extra wrapping via dyn seems odd